content/en/docs/guides/deploying-applications-with-ocm-and-gitops.md

```diff
@@ -0,0 +1,217 @@
+
+Here's a diagram showing what we'll be building:
+
+![deploy-applications-with-gitops](/images/deploy-applications-diagram.png)
+
+As you can see we'll add some manifests to a git repository that will be deployed by Flux. These will, in turn, deploy a resource from an OCM repository, in this case, a `Deployment` of the `podinfo` microservice.
+
+If you'd like to learn how to build a component then checkout our getting started guide [here](/docs/guides/getting-started-with-ocm)
+
+## Table of contents
+- [Requirements](#requirements)
+- [Setup the Environment](#setup-the-environment)
+- [Deploy the OCM Controller](#deploy-the-ocm-controller)
+- [Deploy the Component](#deploy-the-component)
+- [Wrapping Up](#wrapping-up)
+
+### Requirements
+
+- [OCM command line tool](https://github.com/open-component-model/ocm)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [git](https://git-scm.com/downloads)
+- [gh](https://github.com/cli/cli)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [flux](https://fluxcd.io/flux/installation/#install-the-flux-cli)
+
+
+### Setup the environment
+
+First of all, let's create a cluster using `kind`:
+
+```bash
+kind create cluster
+```
+
+With the cluster created, we can now bootstrap Flux to automate the deployment of our component. Flux can create a repository and clone it to our local environment by running the following shell command:
+
+```bash
+export GITHUB_REPOSITORY=podinfo-flux-repo
+
+flux bootstrap github \
+  --owner $GITHUB_USER \
+  --repository $GITHUB_REPOSITORY \
+  --path ./clusters/kind \
+  --personal
+```
+
+This command will create a GitHub repository named `podinfo-flux-repo`, configure Flux to use it, and deploy the resources in the ./clusters/kind directory to our Kubernetes cluster.
+
+Let's now clone the repository flux has created and put in place the manifests required to deploy components:
+
+```bash
+gh repo clone $GITHUB_REPOSITORY && cd $GITHUB_REPOSITORY
+```
+
+We'll add a `Kustomization` to the `./clusters/kind` directory in order to reconcile any resources found in the `./components` directory:
+
+```bash
+cat > ./clusters/kind/components_kustomization.yaml <<EOF
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: components
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  prune: true
+  targetNamespace: ocm-system
+  sourceRef:
+    kind: GitRepository
+    name: flux-system
+  path: ./components
+EOF
+```
+
+Commit this file, push and then ensure Flux has reconciled the resource:
+
+```bash
+git add ./clusters/kind/components_kustomization.yaml
+git commit -m "add components kustomization"
+git push
+
+# trigger an immediate reconciliation of our repo
+flux reconcile source git flux-system
+
+# view kustomizations and their status
+flux get kustomizations
```

Comment:
Mention that this will error at first because the `components` folder is empty and not there yet. So don't panic...

Reply:
```
flux get kustomizations
NAME            REVISION        SUSPENDED       READY   MESSAGE                                                                                    
components                      False           False   kustomization path not found: stat /tmp/kustomization-2542069021/components: no such file or directory
flux-system     main/bcb6754    False           True    Applied revision: main/bcb6754 

```

The output of the above `flux get kustomizations`.

Reply:
maybe i can move it to later in the piece.